tests/func/data/db/test_index.py

```diff
@@ -74,9 +74,9 @@ def unreliable_download(_from_fs, from_info, _to_fs, to_info, **kwargs):
         for from_i, to_i in zip(from_info, to_info):
             on_error(from_i, to_i, Exception())
 
-    mocker.patch("dvc_objects.fs.generic.transfer", unreliable_download)
-    with pytest.raises(DownloadError):
-        dvc.pull()
+    with mocker.patch("dvc_objects.fs.generic.transfer", unreliable_download):
+        with pytest.raises(DownloadError):
+            dvc.pull()
```

Comment:
This took awhile to dig up, but somehow previous mocker was leaking into other tests. I found that it was being used in experiments tests. 🤷 After quite some research I still have no idea why and how, I'm just as shocked as you are.